README.md

```diff
@@ -114,3 +114,23 @@ The interesting fields for this use case are:
   the SSO user ID to be sent to Segment
 - `status.compliantUsername` - Contains the username used in the cluster audit
   log.
+
+## Setting up a containerized Splunk instance
+
+To set up a containerized Splunk instance, you can use either podman or docker. Follow these steps:
+
+1. Run the container with the desired configuration:
+   ```
+   podman run -p 8000:8000 -p 8089:8089 -e "SPLUNK_PASSWORD=YourPassword" -e "SPLUNK_START_ARGS=--accept-license" -it --name splunk_container docker.io/splunk/splunk:latest
+   ```
+   This command maps port 8000 from the container to the host machine, set a password that is a minimum of 8 characters long (the minimum requirement), and accepts the Splunk license.
+
+   For more information about the specific command inputs and options, refer to the [Splunk documentation](https://docs.splunk.com/Documentation/Splunk/9.0.4/Installation/DeployandrunSplunkEnterpriseinsideDockercontainers).
+2. Once the container is running, you can log in to the Splunk instance using the username `admin` and the password you set up.
```

Comment:
The text alignment here seems to be off.

Reply:
Not sure what you refer to?